### DISCUSSION TITLE
Missing break statement in create-remix

### DISCUSSION THREAD

**Discussion by Drylozu** [remix-run/remix#886] [General]
Hi, I didn't know where to write this, but I think a `break` is missing from the create-remix package example.

I don't know if this is intended. The `case` statements are executed from the first one that matches (if `caught.status` is 401, it will execute from `case 401:`) to the `default:` because they don't have a `break`.
Resulting in the same message every time (`There was a problem with your request! 401 Forbidden`, where 401 and Forbidden are `caught.status` and `caught.statusText`).

https://github.com/remix-run/remix/blob/a7712f27002cbedbdf145787216b9fc2f04fd186/packages/create-remix/templates/_shared_js/app/routes/demos/params/%24id.jsx#L55-L75
https://github.com/remix-run/remix/blob/a7712f27002cbedbdf145787216b9fc2f04fd186/packages/create-remix/templates/_shared_ts/app/routes/demos/params/%24id.tsx#L56-L76

Thanks for reading, have a nice day!

**Comment by MichaelDeBoey** [comment]
This is already fixed in the `dev` branch by #611 
https://github.com/remix-run/remix/blob/88754a0a34ccbfcbac3f3d42a441c0f8b74211e6/packages/create-remix/templates/_shared_js/app/routes/demos/params/%24id.jsx#L56-L78
https://github.com/remix-run/remix/blob/88754a0a34ccbfcbac3f3d42a441c0f8b74211e6/packages/create-remix/templates/_shared_ts/app/routes/demos/params/%24id.tsx#L56-L78
